twine/__main__.py

```diff
@@ -25,7 +27,21 @@ def main() -> Any:
     try:
         return cli.dispatch(sys.argv[1:])
     except (exceptions.TwineException, requests.HTTPError) as exc:
-        return "{}: {}".format(exc.__class__.__name__, exc.args[0])
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            "--no-color", default=False, required=False, action="store_true",
+        )
+
+        args, _ = parser.parse_known_args(sys.argv[1:])
+
+        pre_style, post_style = "", ""
+        if not args.no_color:
+            colorama.init()
+            pre_style, post_style = colorama.Fore.RED, colorama.Style.RESET_ALL
+
+        return "{}{}: {}{}".format(
```

Comment:
I think this is a great opportunity to use an f-string for clarity:

```python
        return f"{pre_style}{exc.__class__.__name__}: {exc.args[0]}{post_style}"
```

However, that still buries the "error message" inside of the colorama logic. What if we extracted all of the colorama-related code into a helper method, so the handling could look something like:

```python
    try:
        return cli.dispatch(sys.argv[1:])
    except (exceptions.TwineException, requests.HTTPError) as exc:
        return _format_error(f"{exc.__class__.__name__}: {exc.args[0]}")
```

Reply:
Yeah good idea, I'll add that in my next commit, thanks.